### PR TITLE
[CVE] Improper input validation in PostCSS

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
https://issues.redhat.com/browse/DFBUGS-1168
https://issues.redhat.com/browse/DFBUGS-1167
https://issues.redhat.com/browse/DFBUGS-1166

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -177,18 +177,19 @@
...
     "webpack-dev-server": "^5.1.0"
   },
   "resolutions": {
     "webpack": "^5.96.1",
     "webpack-bundle-analyzer/ws": "^7.5.10",
     "webpack-dev-server/express": "^4.21.0",
     "tough-cookie": "^4.1.3",
-    "cross-spawn": "^7.0.6"
+    "cross-spawn": "^7.0.6",
+    "postcss": "^8.4.49"
```
Comment (alfonsomthd) [inline package.json:192]: Next time put them in alphabetical order, please.

**Comment by GowthamShanmugam** [comment]
release-4.17

**Pull request merged**